datahub-web-react/src/alchemy-components/components/Modal/Modal.tsx

```diff
@@ -57,7 +58,7 @@ const ButtonsContainer = styled.div`
 export interface ModalButton extends ButtonProps {
     text: string;
     key?: string;
-    onClick: () => void;
+    onClick?: () => void;
```

Comment:
curious why make this optional? i feel like buttons should always have a click handler but obviously could be missing something